szperacz/logger.py

```diff
@@ -1,18 +1,36 @@
 
 
 class Logger:
+    global log_level
+    log_level = 0
+    level = log_level
     DEBUG = '[DEBUG]'
     INFO = '[INFO]'
     ERROR = '[ERROR]'
 
+    error = open("error_log.txt", mode='w')
+
+    def set_log_level(self):
+        pass
+
+    def get_log_level(self):
+        return print(f'Logging level: {self.level}')
+
     def debug(self, msg):
-        return print(f'{self.DEBUG} {msg}')
+        if log_level == 0:
```

Comment:
Instead of using global `log_level`, try encapsulating it and using like that: `self.log_level`